var/spack/repos/builtin/packages/py-black/package.py

```diff
@@ -52,6 +52,9 @@ class PyBlack(PythonPackage):
     depends_on("py-ipython@7.8:", when="+jupyter", type=("build", "run"))
     depends_on("py-tokenize-rt@3.2:", when="+jupyter", type=("build", "run"))
 
+    # Needed because this package is used to bootstrap Spack (Spack supports Python 3.6+)
+    depends_on("py-dataclasses@0.6:", when="^python@:3.6", type=("build", "run"))
```

Comment:
Are we supporting anything this would match for anymore? I may be misreading but this only applies below 3.6 right?

Reply:
Nevermind, I misread it, sorry.